ansible/roles/software/ckan/security-fix/tasks/main.yml

```diff
@@ -10,3 +10,21 @@
   with_items:
     - /etc/ckan
     - "{{ project_source_new_code_path }}"
+
+- name: Fix 'Content-Cache' header
+  replace:
+    dest: "{{ project_source_new_code_path }}/src/ckan/ckan/lib/base.py"
+    regexp: ' = "private"'
```

Comment:
```
Currently, we're having Ansible find and replace single lines in the existing sshd_config, which can get messy. The file I looked at on one of the jump boxes has a bunch of duplicate lines, which makes it difficult to follow, and can lead to bugs.

This pull request replaces all of that logic by overwriting the ssd_config from a template. This is a practice I saw recommended by various Ansible users, as it makes the resulting state of the file deterministic/explicit.

Would love early feedback!
```
LGTM as is (lets queue up when/what in another issue for template management) - @dano-reisys @FuhuXia please merge